goldenmaster/Plugins/ApiGear/Source/ApiGearOLink/Private/unrealolink.cpp

```diff
@@ -11,6 +11,24 @@ THIRD_PARTY_INCLUDES_END
 
 DEFINE_LOG_CATEGORY(LogApiGearOLink);
 
+#if (ENGINE_MAJOR_VERSION >= 5)
+FTSTicker& GetCoreTicker()
```

Comment:
to be discussed:
In unity build mode, c++ files are "unified into larger" c++ files for faster compilation.
Therefore functions names should be unique.
We already have similar workarounds in ApiGearConnection and OLinkHostPrivate.
Wondering whether we should have a common ApiGearHelper::GetCoreTicker or so?

Reply:
yes, that's a good idea